### PR TITLE
Remove click from api funcs

### LINKED ISSUE #15: Removing click from commands
Hi,

I am not sure if I should have the discussion here so sorry in advance. Would it be possible to expose the available functions without click? For example, I would like to use the API you have created to write a snippet as the following:

```python
from glacier_upload.upload import upload

upload(...)
```

However, when I currently do this within a python console I receive an issue about the number of arguments. Reading online I found an answer like: https://stackoverflow.com/a/40094408 which may allow both usages as the functions are straightforward.

An example similar to the link would be

```python

# click decorators here
def upload_command(...):
    upload(...) # call the original upload here

def upload(...):
    # stays as it currently is but implements the functionality
```

Let me know your thoughts on this as I would be interested in submitting a pull request for this. My further use case would be having upload() return back the response metadata to store the archive id myself but that will be for another issue.

Great work BTW.

### PR DESCRIPTION
**Pull request by avinash-oza**
Closes #15 . Let me know if you have any alternative ideas otherwise I would be happy if this could be merged

### CODE REVIEW

**Comment by avinash-oza** [comment]
@tbumi any feedback?

File: src/glacier_upload/delete_archive.py [hunk 1]
```diff
@@ -17,3 +12,12 @@ def delete_archive(vault_name, archive_id):
         archiveId=archive_id)
 
     click.echo('Delete archive request sent.')
+
+#TODO: the argument name was wrong and wasnt working before
```
Comment (tbumi) [inline src/glacier_upload/delete_archive.py]: I'm not sure I understand this TODO tag, what is there still not yet done?
By the way thanks for fixing the wrong argument name.

**Comment by tbumi** [comment]
@avinash-oza again, sorry for missing the notification for this PR 🙏 After you address my comment above, I will merge this.

**Comment by avinash-oza** [comment]
Hey @tbumi,

No problem. This is my first pull request so I figured something may have been wrong.

Thanks again!

**Comment by tbumi** [comment]
Thanks for the contribution! Merging this now.

**Pull request merged**